xmlutils-ballerina/build.gradle

```diff
@@ -215,10 +216,26 @@ publishing {
     }
 }
 
-ballerinaBuild.dependsOn ":xmlutils-native:build"
-ballerinaTest.dependsOn unpackJballerinaTools
 ballerinaBuild.dependsOn unpackJballerinaTools
-ballerinaTest.dependsOn ":xmlutils-native:build"
+ballerinaBuild.dependsOn test
+build.dependsOn ballerinaBuild
+
 test.dependsOn ballerinaTest
 ballerinaBuild.dependsOn test
+
+
+ballerinaTest.dependsOn updateTomlVerions
+ballerinaTest.dependsOn initializeVariables
+ballerinaTest.dependsOn ":${packageName}-native:build"
+ballerinaTest.finalizedBy revertTomlFile
+test.dependsOn ballerinaTest
+
+ballerinaBuild.dependsOn test
+ballerinaBuild.dependsOn updateTomlVerions
+ballerinaBuild.dependsOn initializeVariables
+ballerinaBuild.dependsOn ":${packageName}-native:build"
+ballerinaBuild.finalizedBy revertTomlFile
 build.dependsOn ballerinaBuild
```

Comment:
This again is duplicated. Shall we check these dependencies and remove the duplicates?